README.md

```diff
@@ -1,8 +1,62 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Marissa Schmitt . Jarvis Consulting
 
+I am a recent graduate from Conestoga College where I studied Software Engineering Technology. I am excited to be challenged by new technologies and expand my skillset as I have learned a variety of programming languages during the duration of my program. During my time at Conestoga College, I learned about web development, mobile development, relational databases, object-oriented programming, and assembly language. My interest in software engineering started in high school where I pursued many computer science and computer engineering courses. I am intrigued by all aspects of software engineering whether it be frontend, backend, or working with a database directly. Currently I am working as a software developer/data engineer at Jarvis to improve my skills by learning about core Java programming, design patterns, Maven, and agile development.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** C++, C#, C, JavaScript (Vue and ReactJS), MongoDB
+
+**Familiar:** .NET, Swift, Kotlin, Assembly language, Waterfall Methodology
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_MarissaSchmitt/tree/master/linux_sql)]: Implemented a Linux cluster monitoring agent to manage and insert resource usage information into a PostgreSQL database. The host usage information was collected through bash commands and automated using crontab.
```

Comment:
Implemented a Linux cluster monitoring agent to manage and insert resource data, such as CPU, RAM, DISK usage into a PostgreSQL database. The host usage information was collected through bash commands and automated using crontab.

Docker, SQL, data analytics.